src/codegen/llvm/codegen_amdgpu.cc

```diff
@@ -135,9 +135,15 @@ runtime::Module BuildAMDGPU(Array<LoweredFunc> funcs, std::string target) {
   CHECK(target.length(
 ) >= 4 &&
         target.substr(0, 4) == "rocm");
+  TVMContext tvmCtx;
+  tvmCtx.device_type = kROCM;
+  tvmCtx.device_id = 0;
+  TVMRetValue val;
+  tvm::runtime::DeviceAPI::Get(tvmCtx)->GetAttr(tvmCtx, tvm::runtime::kComputeVersion, &val);
```

Comment:
To be safe, I would first query tvm::runtime::DeviceAPI::Get(tvmCtx)->GetAttr(tvmCtx, tvm::runtime::kExist, &val) to check if device exists, then if it exists, append the gfx arch string, otherwise assign an default